gemrb/core/Map.cpp

```diff
@@ -1395,8 +1395,11 @@ void Map::Shout(Actor* actor, int shoutID, unsigned int radius)
 			continue;
 		}
 
-		if (radius) {
-			if (Distance(actor->Pos, listener->Pos)>radius) {
+		if (!global) {
+		        const Point& A = actor->Pos;
```

Comment:
Mixed indentation.

Reply:
line number?

Reply:
1399. No spaces should be used for indentation anywhere.